scripts/maintenance/generate_audit_summary.py

```
@@ -0,0 +1,94 @@
+        # Comment on the PR if it exists and GH CLI is available
+        branch = item['branch']
+        if branch in pr_map:
+            pr_num = pr_map[branch]
+            comment = '### ⚠️ Systemic Regressions Detected\n\nThis branch is currently blocked by the following regressions:\n\n'
+            for issue in issues:
+                comment += f'- {issue}\n'
+            comment += '\nPlease restore these critical files or security helpers before merging.'
+            run_command(['gh', 'pr', 'comment', str(pr_num), '--body', comment])
+
+    # Generate Markdown Summary
+    date_str = datetime.now().strftime('%Y-%m-%d')
```

Ruff (0.15.10)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)